app/gui2/src/components/CircularMenu.vue

```diff
@@ -64,6 +70,47 @@ const emit = defineEmits<{
   }
 }
 
+.CircularMenuFull {
+  &:before {
+    content: '';
+    position: absolute;
+    clip-path: path(
+      evenodd,
+      'M0,52 A52,52 0,1,1 104,52 A52,52 0,1,1 0, 52 z m52,20 A20,20 0,1,1 52,32 20,20 0,1,1 52,72 z'
+    );
+    backdrop-filter: var(--blur-app-bg);
+    background: var(--color-app-bg);
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.More {
+  position: absolute;
+  left: -5.5px;
+  top: 27px;
+  width: 42px;
+  height: 17px;
+  clip-path: path(
+    evenodd,
+    'M7.96503 8C7.96503 3.58172 11.5467 0 15.965 0H26.035C30.4533 0 34.035 3.58172 34.035 8V20 H7.96503V8Z'
+  );
+  transform: scale(0.8);
+  backdrop-filter: var(--blur-app-bg);
+  background: var(--color-app-bg);
+  z-index: -2;
+
+  &:after {
+    content: '...';
+    font-size: 15px;
+    display: block;
+    text-align: center;
+    z-index: 10000;
+    margin-top: -10px;
+    opacity: 0.3;
+  }
```

Comment:
Cannot these three dots be just a content of `.More`? And I don't see any usage of this class.